### PR TITLE
feat: add code of conduct

### PR DESCRIPTION
**Pull request by schmelto**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+<insert here>.
```
Comment (schmelto) [inline CODE_OF_CONDUCT.md:63]: here an email-address is missing what email address should be used?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Brittle

**Comment by TheKicker** [comment]
@Difint - I'm guessing the same email account I invited to the Google Analytics reports?

**Comment by Difint** [comment]
@TheKicker please use luncosim@gmail.com

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (schmelto) [inline CODE_OF_CONDUCT.md:63]:
```suggestion
luncosim@gmail.com.
```

**Comment by schmelto** [comment]
Done ✅

**Comment by Difint** [comment]
@schmelto Thank you so much for you help!

**Pull request merged**